src/index.ts

```diff
@@ -243,6 +243,4 @@ function Steppp(element: HTMLElement, options: any = defaultOptions) {
   };
 }
 
-Steppp.stepIsValid = (_slide: HTMLElement): boolean => true;
-
```

Comment:
Not actually used anywhere... I think this was carried over from the cowboy version we had built in the past.